things/service_test.go

```diff
@@ -928,6 +925,15 @@ func TestListChannelsByThing(t *testing.T) {
 			size: n - chsDisconNum,
 			err:  nil,
 		},
+		"list all channels by existing thing with no limit": {
```

Comment:
Do we have `list all non connected channels by existing thing with no limit` test?